omnibus/omnibus-test.ps1

```diff
@@ -1,40 +0,0 @@
-# Stop script execution when a non-terminating error occurs
```

Comment:
This'll get replaced with verify-install.ps1 that is equivalent to verify-install.sh